dependencies.xml

```diff
@@ -58,6 +58,7 @@ Note all install methods after "main" take
     <line_profiler optional='True'/>
     <!-- <ete3 optional='True'/> -->
     <pywavelets optional='True'>1.1</pywavelets>
+    <fmpy optional='True' source='forge'/>
```

Comment:
This will not work unless library_handler is patched:
```diff
diff --git a/scripts/library_handler.py b/scripts/library_handler.py
index 5bd0052ae..c9200f6d4 100644
--- a/scripts/library_handler.py
+++ b/scripts/library_handler.py
@@ -584,7 +584,7 @@ if __name__ == '__main__':
       elif args.subset == 'forge':
         # take libs from conda-forge
         src = '-c conda-forge '
-        addOptional = False
+        addOptional = args.addOptional
         limit = ['forge']
       elif args.subset == 'pip':
         src = ''
```

Reply:
fixed